UDPServer/SiceUDPServer.cs

```diff
@@ -27,10 +27,12 @@ public class SiceUDPServer : IServerInterface, IHandle<ServerCommandEvent>
         public bool HasLoggedIn { get; set; }
         public int ConnectedControllerID { get; set; }
         public string ConnectedControllerInfo { get; set; }
+        public ServerContextRepo ContextRepo { get; set; }
 
-        public SiceUDPServer(IEventAggregator eventAggregator)
+        public SiceUDPServer(IEventAggregator eventAggregator, ServerContextRepo contextRepo)
```

Comment:
Use full names, repository instead of Repo. It's important because it follows a convention that's part of the IT industry.